web_s2i/application.py

```diff
@@ -1,17 +1,29 @@
-import sys
+#!/usr/bin/env python3
 
 from flask import Flask, flash, render_template, redirect, url_for, request
 from flask_mysqldb import MySQL
 from wtforms import Form, StringField, PasswordField, validators
 from passlib.hash import sha256_crypt
+import yaml
+
+
+def load_config(config_file):
+    with open(config_file, 'r') as stream:
+        try:
+            return yaml.safe_load(stream)
+        except yaml.YAMLError as exc:
+            print(exc)
+
+
+cfg = load_config('configuration.yml')
```

Comment:
here it is cfg, in other files it is CONFIGURATION. Be consistent.

Reply:
fixed